api/src/utils/query-actions.js

```diff
@@ -28,8 +28,8 @@ exports.publish = async function(obj) {
       obj.datePublished = date;
       obj.markModified('datePublished');
 
-      obj.dateUpdated = date;
-      obj.markModified('dateUpdated');
```

Comment:
I think we want to keep this level of detail.  Same for unPublish

Reply:
My thoughts at the time, were around whether or not we consider toggling the publish state as an update/edit or not.  But I could see the argument either way.  Can put it back.